e2e/features/submit-appeal.feature

```diff
@@ -2,9 +2,22 @@ Feature: Submit appeal application
 
   Background:
     Given I am signed in as a `Legal Rep`
+    Then I wait for 10 seconds
     And I create a new case
     And I save my initial appeal
 
+  @xbrowsertest
+  Scenario: Submit an appeal application
+
```

Comment:
This scenario only covers submitting the appeal, we may add more scenarios to cover E2E

Reply:
> This scenario only covers submitting the appeal, we may add more scenarios to cover E2E going forward

Agreed. Any new scenarios for which we need crossbrowser testing needs to be tagged with 'xbrowsertest'